contribute.html

```diff
@@ -13,20 +13,92 @@ <h1 class="p-heading--stylized u-no-margin--bottom">Contribute</h1>
+      <p>When <a href="https://github.com/vanilla-framework/vanilla-framework/issues/new" class="p-link--external">submitting a new issue</a>, please check that it hasn't already been raised by someone else. We've provided a template for new issues which will help you structure your issue to ensure it can be picked up and actioned easily.</p>
       <p>Please provide as much information possible detailing what you're currently experiencing and what you'd expect to experience.</p>
-      <p>To work on an issue, please fork this repo and create a new branch on your local fork. When you're happy and would like to propose that changeset to be merged back upstream, open a pull request to merge from your local <code>origin/master</code> to <code>upstream/master</code>.</p>
+      <p>To work on an issue, please fork this repo and create a new branch on your local fork. When you're happy and would like to propose that changeset to be merged back upstream, open a pull request to merge from your local origin/master to upstream/master.</p>
       <p>When committing changes, make sure to group related changes so that the project is always in a working state. Use succinct yet descriptive commit messages to allow for easy reading of the commit log.</p>
-      <h2>Running tests locally</h2>
-      <p>Before proposing a pull request, ensure that the tests pass on your local fork. To kick off the tests, run <code>./run test</code>.</p>
-      <h2>Licences</h2>
-      <p>The content of this project is licensed under the <a class="p-link--external" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International license</a>, and the underlying code used to format and display that content is licensed under the <a class="p-link--external" href="http://opensource.org/licenses/lgpl-3.0.html">LGPLv3</a> by <a class="p-link--external" href="http://www.canonical.com/">Canonical Ltd.</a></p>
-      <p>Last updated: <time>{{ page.sitemap.lastmod | date: '%d %B %Y'}}</time></p>
+    </div>
+    <div class="col-4">
+      <div class="p-card is-bordered">
+        <h3>File a request/bug</h3>
+        <p>We use <a href="https://github.com/vanilla-framework/vanilla-framework/issues" class="p-link--external">GitHub issues</a> to track all our bugs and feature requests.</p>
+      </div>
+      <div class="p-card is-bordered">
+        <h3>Chat with us</h3>
+        <p>Find out about new releases, latest features and get help on <a href="https://twitter.com/vanillaframewrk" class="p-link--external">Twitter</a>.</p>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="p-strip is-shallow">
+  <div class="row">
+    <hr>
```

Comment:
I think you can get rid of the strip div and just have the row div (here and the one after the team), otherwise it feels a bit too space-y

Reply:
@Caleb-Ellis how? If I remove the strip, the <hr> goes full-width and if I put it in one of the strips then it's not aligned vertically... and if I make them all one strip then there's not enough space between rows. How, Caleb, HOW?!

Reply:
Replace
``` html
<div class="p-strip is-shallow">
  <div class="row">
    <hr>
  </div>
</div>
```
with
```html
<div class="row">
  <hr>
</div>
```